extensions/spark/kyuubi-spark-connector-tpcds/src/test/resources/tpcds_2_4/q1.sql

```diff
@@ -0,0 +1,36 @@
+--
```

Comment:
I copy them from `dev/kyuubi-tpcds`, I'm not sure if that module can be replaced by this connector in the future.